### PR TITLE
Refuse to install on Python 2

### PR DESCRIPTION
**Pull request by adisbladis**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -37,5 +44,5 @@
        package_data         = {'': ['*.ui']},
        license              = 'GPL',
        zip_safe             = False,
-       install_requires     = [ 'scapy', 'dpkt', 'NetfilterQueue', 'psutil' , 'pyinotify']
+       install_requires     = [ 'scapy', 'dpkt', 'NetfilterQueue', 'psutil' , 'pyinotify', 'PyQt5']
```
Comment (melizeche) [inline setup.py]: "setup.py install" fails to install scapy with python 3, `'scapy-python3'` worked for me

Comment (adisbladis) [inline setup.py]: Added to this PR, thanks! 👍

**Comment by melizeche** [comment]
Worked like charm 👌

**Comment by in1t3r** [comment]
It does hard depend now on the python3 I made changes to the readme file
and submited issue and pull request either way those changes should be
reflected in documentation.

So it does depend now on python3-dev on python3-pyqt5 and of course
python3-setuptools.

**Pull request merged**